qa/tasks/fs.py

```diff
@@ -67,6 +68,11 @@ def post_upgrade_checks(ctx, config):
             old_status = mdsc.status(epoch=i)
             old_fs = old_status.get_fsmap(fscid)
             old_mdsmap = old_fs['mdsmap']
+            if fs_state['max_mds'] > 1 and \
```

Comment:
I'm not sure how adding debug log message actually tests that max_mds was reduced or not reduced to 1 when "mgr/orchestrator/fail_fs" was set to False or True respectively.

I think you should instead edit line 63,
```
should_decrease_max_mds = fs_state['max_mds'] > 1
```
to
```
# pseudo code:
should_decrease_max_mds = (fs_state['max_mds'] > 1) and (mgr/orchestrator/fail_fs is False)
```

Can't the assertions in lines 82 and 83 be simplified as,
```
assert should_decrease_max_mds == did_decrease_max_mds
assert should_disable_allow_standby_replay == did_disable_allow_standby_replay
```

Reply:
Right, I'll update the checks here. Thanks for pointing this out.